gemini_test.py

```diff
@@ -58,7 +58,7 @@ def test_load_random_with_nemesis(self):
         sleep_before_start = float(self.params.get('test_duration')) * 60 * .1
         self.log.info('Sleep interval {}'.format(sleep_before_start))
         time.sleep(sleep_before_start)
-
+        self.run_post_prepare_cql_cmds()
```

Comment:
(method name is probably not ideal here)

Reply:
so please improve it

Reply:
@fgelcer , how about `post_initial_stress_cql_cmds` for the yaml parameter and `run_post_initial_stress_cql_cmds` for this method?

Reply:
that the name it was given in longevity, why the name here isn't good ? (since gemini doesn't have prepare stage ?)

Reply:
> (method name is probably not ideal here)

why do you think the name is not ideal?

Reply:
@fruch , right, that's the point. but i can take my words back if you think it's good enough this way as well.